sdk/runanywhere-python/runanywhere/server/app.py

```
@@ -0,0 +1,643 @@
+            sample_rate, samples = decode_wav(raw)
+        except Exception as exc:  # noqa: BLE001
+            raise HTTPException(status_code=400, detail=f"could not decode audio (send WAV): {exc}")
+        if sample_rate != STT_SAMPLE_RATE:
+            samples = downsample(samples, sample_rate, STT_SAMPLE_RATE)
+        pcm16 = pcm16_bytes(samples)
+        async with lock:
+            text = await stt.atranscribe(pcm16)
+        return {"text": text}
+
+    @app.post("/v1/audio/speech", dependencies=guarded)
+    async def speech(req: SpeechRequest, mgr: ModelManager = Depends(get_manager)):
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)